lib/galaxy/tools/__init__.py

```diff
@@ -2223,6 +2222,19 @@ def handle_input(
             collection_info=collection_info,
             completed_jobs=completed_jobs,
         )
+
+        tags = incoming.get("tags", [])
```

Comment:
Can you document this in https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/galaxy/api/tools.py#L771